notebooks/harmony_concatenation/Harmony_Concatenation.ipynb

```diff
@@ -2,839 +2,294 @@
  "cells": [
```

Comment:
<p>I'm pretty sure harmony-py will look up the .netrc file if it exists so we shouldn't need explicit login info</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/podaac/tutorials/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>